test/integration/src/main/java/io/pravega/test/integration/auth/customplugin/CustomAuthHandler.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.integration.auth.customplugin;
```

Comment:
If this is for test purposes, what about calling it `TestAuthHandler` rather than `CustomAuthHandler`?

Reply:
Sure, I've modified the class names accordingly.